packages/odyssey-storybook/src/components/odyssey-mui/Link/Link.stories.tsx

```diff
@@ -135,5 +137,15 @@ export const External: StoryObj<LinkProps> = {
     children: "Visit okta.com",
     rel: "noopener",
     target: "_blank",
+    ariaLabel: "external-link",
+  },
+  play: async ({ canvasElement, step }) => {
+    await step("Link Aria-Label", async ({ args }) => {
+      const canvas = within(canvasElement);
+      const link = canvas.getByLabelText("external-link");
+      expect(link).toHaveAttribute("href", args.href);
+      expect(link).toHaveAttribute("rel", args.rel);
+      expect(link).toHaveAttribute("target", args.target);
+    });
```

Comment:
What do you think about this?

```suggestion
    ariaLabel: "External Link",
  },
  play: async ({ canvasElement, step }) => {
    await step("Link Aria-Label", async ({ args }) => {
      const canvas = within(canvasElement);
      const link = canvas.getByRole("link", { name: "External Link" });
      expect(link).toHaveAttribute("href", args.href);
      expect(link).toHaveAttribute("rel", args.rel);
      expect(link).toHaveAttribute("target", args.target);
    });
```

I prefer using `getByRole` rather than `getByLabelText` because it makes it clear our intent is to get a link, not some random HTML element on the page. Also helps with type safety.